.github/workflows/lint-test.yml

```diff
@@ -37,8 +37,12 @@ jobs:
   go_test:
     uses: 'abcxyz/pkg/.github/workflows/go-test.yml@main' # ratchet:exclude
     strategy:
+      fail-fast: false
```

Comment:
This is required - otherwise the linux tests failing will cancel any in-flight tests on windows.

Reply:
👍 

Perhaps add this comment as a comment in the file for our descendants.